docker/start.py

```diff
@@ -305,13 +305,19 @@ def main():
     logger.debug("URL_ROOT = {}".format(url_root))
     logger.debug("URI = {}".format(uri))
 
-    # default period for reindexing (in minutes)
-    reindex_env = os.environ.get('REINDEX')
-    if reindex_env:
-        reindex_min = reindex_env
-    else:
-        reindex_min = 10
-    logger.debug("reindex period = {} minutes".format(reindex_min))
+    # default period for syncing (in minutes)
+    sync_period = 10
+    sync_env = os.environ.get('SYNC_TIME_MINUTES')
+    if sync_env:
+        try:
+            n = int(sync_env)
+            if n > 0:
```

Comment:
This should be `n >= 0` to allow for a zero value (and bypassing the sync loop)

Reply:
will fix, thanks !